common/lib/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ endif()
 
 if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     target_sources(villas-common PRIVATE
-        kernel/pci.cpp
+        kernel/devices/pci_device.cpp
```

Comment:
Would it be okay, if we rename it to `kernel/devices/pci.h`? I dislike the duplication of `device` in the path.

Reply:
Okay, good argument. Lets keep `pci_device.hpp` then